### PR TITLE
Update to Deno v0.41

### PR DESCRIPTION
**Pull request by Soremwar**
This fixes Oak under Deno v0.41

### CODE REVIEW

**Comment by Soremwar** [comment]
Still WIP

File: send.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ function isHidden(root: string, path: string) {
 
 async function exists(path: string): Promise<boolean> {
   try {
-    return (await Deno.stat(path)).isFile();
+    return Deno.stat(path).then(x => x.isFile);
```
Comment (ronhippler) [inline send.ts:62]: the try/catch block is not used without await. Do not use .then for this case.

Comment (Soremwar) [reply send.ts]: Taking note
...
Done

**Comment by tubbo** [comment]
Is there a way to "try this out" by loading the module from github? I tried to load `mod.ts` like this from your GitHub, but Deno still seems to be going to the deno.land URL:

```javascript
import { Application as Oak } from "https://raw.githubusercontent.com/Soremwar/oak/v0.41/mod.ts";
```

Like most of us, I'm pretty new to Deno so I'm not sure how this is supposed to work.

**Comment by kitsonk** [comment]
Apologies for not taking a look at this earlier.  I have gone through and done quite a few updates to bring oak current with 0.42.0 and will do more work tomorrow and release a tag.

**Pull request closed without merging**